### DISCUSSION TITLE
combine proxied request statistics into one statistic ?

### DISCUSSION THREAD

**Discussion by zoe-jobson-IL** [artilleryio/artillery#2440] [Artillery]
Our app creates a semi-unique endpoint for each user accessing a particular area - they look like this
https://79249.proxy-http.path.com/webapp/auth/login
where each user has a different number, We are using playwright scripts (previously created for e2e testing with a few slight changes) with artillery to test our app. I would like to either combine these into one stat for each type of endpoint e.g. proxy-http.path.com/webapp/auth/login, or filter them out entirely (we are using steps so have user type version of these requests (e.g. users opens code editor))

**Comment by hassy** [comment]
hi @zoe-jobson-IL 👋 This isn't possible right now. Is this what you'd like to be able to do (making sure I understand):

* Each virtual user is going to be accessing the app on a different URL unique to that user
* Requests are all actually served by the same backend so you want to be able to tell Artillery to map those unique URLs to one identifier (e.g. `proxy-http.path.com/webapp/auth/login`)

Also - are you ok to filter those URLs out completely as an alternative because step-specific metrics provide the performance stats you need? (`browser.step.<stepName>` ones in the Artillery report)

**Reply by zoe-jobson-IL** [reply]
yes correct on what I would like to be able to do, and yes for now its OK to filter them out completely, as an example run in your cloud, which had 100 users over 10 fargate machines the report is like this
https://app.artillery.io/load-tests/t6enp_zpgewwk977a7jgftjhgr3dphdcj8j_6c7a/reports
you can guess that when we get up to 1000s of users (or even 10000s) the urls are going to get rather overwhelming and the way things are we can't easily track behaviour over time for any of those urls, though as I said, certainly for now, the steps we have defined are good enough.

**Reply by hassy** [reply]
@zoe-jobson-IL have a prototype of a fix for this working, will publish a PR + canary release later this morning if you'd like to try it with one of your tests

**Reply by hassy** [reply]
@zoe-jobson-IL This is out in a canary release now. To try, install it with:

```
npm install -g artillery@2.0.6-f5cba1a
```

PR #2516 has an example of how it works. Let me know if it does what you need
